### ISSUE TITLE
Maybe add docstring link on startup / in the tutorial to explain what a buffer is

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#971]
Thoughts?

**Comment by jmercouris** [comment]
We can have a terminology section at the beginning of the tutorial, would that work?

**Comment by Ambrevar** [comment]
We already do, right after the Quickstart keys.
The problem is that the quickstart keys make use of the term "buffer".

Maybe just add a "See below for an explanation on 'buffers'" at the beginning?

**Comment by jmercouris** [comment]
Maybe we can have all terminology in the manual hyperlinked to a dictionary. So if you click 'buffer' in the manual, it takes you to the definition of buffer. What do you think?

**Comment by Ambrevar** [comment]
Wouldn't this dictionary just be the class descriptions then?
We could add a link to the buffer class description, it would avoid redundancies.

**Comment by jmercouris** [comment]
Yes and no. The class description is useful, but I was thinking something more high level for a beginner. Like: "Buffers are what Nyxt calls "Tabs" in other browsers".

**Comment by Ambrevar** [comment]
Is there anything other term that can be explained that briefly?

**Comment by jmercouris** [comment]
I don't understand the question, can you please rephrase?

**Comment by Ambrevar** [comment]
Sorry: is there any term beside "buffer" which you think would belong to
such a dictionary?

**Comment by jmercouris** [comment]
There are other such terms, yes. For example, "mode" or "command".

**Comment by aadcg** [comment]
This will be closed by #2811.
